lib/index.js

```diff
@@ -3,7 +3,7 @@ var path = require('path')
 var filewatcher = require('filewatcher')
 var ipc = require('./ipc')
 var resolveMain = require('./resolveMain')
-
+var minimatch = require("minimatch")
```

Comment:
Minor: Please move this with the other `require`s, ideally in :abc: order :smile: